mmdet/evaluation/metrics/crowdhuman_metric.py

```diff
@@ -0,0 +1,914 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import json
+import os.path as osp
+import tempfile
+from collections import OrderedDict
+from multiprocessing import Process, Queue
+from typing import Dict, List, Optional, Sequence, Union
+
+import numpy as np
+from mmengine.evaluator import BaseMetric
+from mmengine.logging import MMLogger
+from scipy.sparse import csr_matrix
+from scipy.sparse.csgraph import maximum_bipartite_matching
+from tqdm import tqdm
+
+from mmdet.registry import METRICS
+
+PERSON_CLASSES = ['background', 'person']
+
+
+@METRICS.register_module()
+class CrowdHumanMetric(BaseMetric):
+
+    def __init__(self,
+                 ann_file: str,
+                 metric: Union[str, List[str]] = 'AP',
```

Comment:
suggest computing all metrics as default
```suggestion
                 metric: Union[str, List[str]] = ['MR', 'AP', 'JI'] ,
```